drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodListDialogFragment.kt

```diff
@@ -112,11 +112,11 @@ class PaymentMethodListDialogFragment : DropInBottomSheetDialogFragment(), Payme
 
         // Check some specific payment methods that don't need to show a view
         when {
-            paymentMethod.type == PaymentMethodTypes.GOOGLE_PAY -> {
+            paymentMethod.type == PaymentMethodTypes.GOOGLE_PAY || paymentMethod.type == PaymentMethodTypes.GOOGLE_PAY_LEGACY -> {
```

Comment:
Or should we make `GooglePayComponent.PAYMENT_METHOD_TYPES` public and use `GooglePayComponent.PAYMENT_METHOD_TYPES.contains(paymentMethod.type)` instead?

Reply:
Yeah that sounds good. I thought we had moved the logic to see if it's supported to the `PROVIDER`, so I was gonna suggest to add this check there as well, but apparently we didn't.